web/src/containers/Sidebar.js

```diff
@@ -197,18 +197,22 @@ class Sidebar extends Component {
 
     const { activeSection } = this.props;
 
+    const hasImage = ['as', 'gu', 'mp', 'vi'].indexOf(place.id) < 0;
+
     return (
       <div className="ds-l-col--3">
         <aside className="site-sidebar">
           <div className="xs-hide sm-hide site-sidebar__sticky">
             <div className="flex items-center ds-u-border-bottom--1 ds-u-padding-y--2 ds-u-margin-bottom--4">
-              <img
-                src={`/static/img/states/${place.id}.svg`}
-                alt={place.name}
-                className="align-middle mr2"
-                width="40"
-                height="40"
-              />
+              {hasImage && (
+                <img
+                  src={`/static/img/states/${place.id}.svg`}
+                  alt={place.name}
+                  className="align-middle mr2"
```

Comment:
While we're in here, can we rip this out? I think this should do the trick (after fiddling with it, it appears that the `align-middle` class doesn't actually have an effect on the way it's centered):
```suggestion
                  className="ds-u-margin-right--2"
```